.github/workflows/ci-nym-vpn-app-macos.yml

```
@@ -0,0 +1,99 @@
+      - name: Setup sccache
+        uses: ./.github/actions/setup-sccache
+
+      - name: Dotenv Action
+        uses: xom9ikk/dotenv@v2.4.0
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ env.RUST_VERSION }}
+          components: rustfmt, clippy
+          targets: ${{ matrix.target }}
```

actionlint (1.7.12)

[error] property "target" is not defined in object type {}

(expression)